api/serializers.py

```diff
@@ -1,9 +1,23 @@
 from rest_framework import serializers
 
 from banks.models import Bank
+from users.models import User
+from accounts.models import Account
 
 
 class BanksSerializer(serializers.ModelSerializer):
     class Meta:
         model = Bank
         fields = '__all__'
+
+
+class UsersSerializer(serializers.ModelSerializer):
```

Comment:
class names should be generally singular
`UserSerializer` or `UserListSerializer` for multiples